packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/siw.md

```diff
@@ -0,0 +1,90 @@
+### 1: Initiate password recovery
+
+To begin the password recovery flow, the user must
+
+1. Click the **Forgot Password?** link on the login page.
+2. Enter their **Email or Username** in the textbox and click **Next**.
+3. Choose **Email** as the authenticator to be used for password recovery and click **Submit**
+
+Okta then tells the user to either click the link in the email or enter the code to continue and sends an email to their email address matching the Forgot Password template that was altered earlier.
+
+<div class="common-image-format">
+
+![Screenshot of email sent to user](/img/advanced-use-cases/custom-pwd-recovery-custom-email.png)
```

Comment:
see my comment above about updating the image description.

Reply:
fixed